content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -151,7 +151,7 @@ exit
 
 {{< feature-state for_k8s_version="v1.20" state="beta" >}}
 
-By default, Kubernetes recursively changes ownership and permissions for the contents of each
+Support for ConfigurableFSGroupPolicy is now enabled by default. By default, Kubernetes recursively changes ownership and permissions for the contents of each
```

Comment:
`{{< feature-state for_k8s_version="v1.20" state="beta" >}}` should already convey that; we don't need to repeat that it's beta.